pkgs/development/compilers/openjdk/openjfx/20.nix

```diff
@@ -93,14 +93,14 @@ in makePackage {
 
   postFixup = ''
     # Remove references to bootstrap.
-    export openjdkOutPath='${openjdk19_headless.outPath}'
+    export openjdkOutPath='${openjdk20_headless.outPath}'
     find "$out" -name \*.so | while read lib; do
       new_refs="$(patchelf --print-rpath "$lib" | perl -pe 's,:?\Q$ENV{openjdkOutPath}\E[^:]*,,')"
       patchelf --set-rpath "$new_refs" "$lib"
     done
   '';
 
-  disallowedReferences = [ openjdk17_headless openjdk19_headless ];
+  disallowedReferences = [ openjdk17_headless openjdk20_headless ];
```

Comment:
Can we explicitly mark this version as out-of-date?

Reply:
Right now it refuses to evaluate because openjdk20 is marked as insecure. The reason why I changed the derivation to use openjdk20 to match was so that if someone wanted to use openjfx20, they just need to mark openjdk20 as allowed instead of needing to specifically allow both.

I can explicitly mark them as insecure if wanted though. Will check how the older EOL versions do it.

Reply:
I think marking this explicitly as insecure helps us when we eventually want to do a clean-up.

Reply:
True. Will make sure all of the EOL versions are marked then.